src/sfdxFlags.ts

```diff
@@ -732,8 +737,10 @@ export function buildSfdxFlags(
       if (!isKeyOf(optionalBuiltinFlags, key)) {
         throw messages.createError('error.UnknownBuiltinFlagType', [key]);
       }
+      // @ts-ignore
       output[key] = optionalBuiltinFlags[key](flag);
```

Comment:
ex: if you already set `options. targetusername `, then this would do that work again?  Is that so that you can override the default `targetusername` with your own flag and have it win?